cargo-shuttle/src/client.rs

```diff
@@ -254,11 +254,25 @@ impl ShuttleApiClient {
             .await
             .context("Failed parsing logs. Is your cargo-shuttle outdated?")
     }
+    pub async fn get_deployment_logs_beta(
+        &self,
+        project: &str,
+        deployment_id: &str,
+    ) -> Result<Vec<LogItemBeta>> {
```

Comment:
Can we return a JSON here instead of lines directly ? Same for the project logs actually. Something like this for now, but that we could easily change to add pagination for instance:
```rust
struct {
  logs: Vec<LogItemBeta>,
}
```